frontend/src/metabase/common/hooks/use-search-list-query/use-search-list-query.unit.spec.tsx

```diff
@@ -28,6 +48,15 @@ const TestComponent = () => {
       {data.map(item => (
         <div key={item.id}>{item.name}</div>
       ))}
+      <div data-testid="metadata">
+        {metadata && Object.keys(metadata).length > 0
+          ? Object.entries(metadata).map(([key, value]) => (
```

Comment:
Could we render actual values we're going to use to make the test more readable? E.g. if you're going to use `available_models` or `total` let's do that explicitly.